docs/index.html

```diff
@@ -1639,7 +1647,9 @@ <h1 class="display-4 theme-content-element-title">Tags</h1>
 		<span class="theme-content-element-source-label">Source:</span>
 		<span class="theme-content-element-source-path">tags.md</span>
 	</span>
-	<div class="theme-content-element-description"><p>Tags can be grouped together and are useful for displaying metadata and filtering.</p>
+	<div class="theme-content-element-description"><p>Tags are interactive elements presented with a keyword or phrase that allows the user to move quickly to the selected point. Tags can be grouped together and are useful for filtering.</p>
+<h3>✅ When to use:</h3><p>Tags are used to help the user find content in the same category.</p>
```

Comment:
Det sier ikke om når tags elemente skal brukes, bare hva tags er gennerelt. Mulig det er greit, var bare litt rart

Reply:
Jag har inget speciellt use case i mina designfiler hur tags skal tas i bruk mer än för den filterfunktionen som finns i DROPS. Kanske håller för nu vad tags är och så kan vi komma upp mer hur det ska brukes senare när vi vet mer?

Reply:
"Use tags to help the user categorize content"